mlir/lib/Dialect/MIOpenOps/Driver/mlir-miopen-driver.cpp

```diff
@@ -29,13 +29,19 @@
 #include "llvm/Support/ToolOutputFile.h"
 #include "llvm/Support/raw_ostream.h"
 
+#include <iostream>
+
 using namespace llvm;
 using namespace mlir;
 
 static cl::opt<std::string> outputFilename("o", cl::desc("Output filename"),
                                            cl::value_desc("filename"),
                                            cl::init("-"));
 
+static cl::opt<std::string>
+    convFlavor("conv_flavor", cl::desc("Convolution Flavor"),
```

Comment:
"flavor" is a bad nomenclature. Suggest to be consistent with MIOpen nomenclature.

Reply:
Agreed. Now on a second look, this field is either `conv2d` or `conv2d_bwd_data`, which matches to the operation name. Maybe call this as `operation`?